<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>271</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/tow_resources.qrc">
    <normaloff>:/icons/tow_icon2.png</normaloff>:/icons/tow_icon2.png</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDockWidget" name="dock_jog">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="floating">
     <bool>false</bool>
    </property>
    <property name="features">
     <set>QDockWidget::NoDockWidgetFeatures</set>
    </property>
    <property name="allowedAreas">
     <set>Qt::BottomDockWidgetArea</set>
    </property>
    <property name="windowTitle">
     <string>Jog Motion</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <widget class="QPushButton" name="pbJogPendant">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Pendant</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbJogPlus">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Jog Left</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons/tow_resources.qrc">
        <normaloff>:/icons/back.png</normaloff>:/icons/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pbJogMinus">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Jog Right</string>
      </property>
      <property name="icon">
       <iconset resource="icons/tow_resources.qrc">
        <normaloff>:/icons/forward.png</normaloff>:/icons/forward.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>287</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuTraverse_Offset">
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Traverse Offset</string>
     </property>
     <addaction name="traverseOff"/>
     <addaction name="traverse1m"/>
     <addaction name="traverse2m"/>
     <addaction name="traverse3m"/>
     <addaction name="traverse4m"/>
     <addaction name="traverseCustom"/>
    </widget>
    <widget class="QMenu" name="menuTwitter">
     <property name="title">
      <string>Twitter</string>
     </property>
     <widget class="QMenu" name="menuAutomatic_Tweeting">
      <property name="title">
       <string>Automatic Tweeting</string>
      </property>
      <addaction name="actionOn_PTP_Move"/>
      <addaction name="actionOn_Jogging"/>
      <addaction name="actionOn_Enable"/>
     </widget>
     <addaction name="menuAutomatic_Tweeting"/>
    </widget>
    <addaction name="menuTraverse_Offset"/>
    <addaction name="menuTwitter"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWiki"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHoming">
    <property name="title">
     <string>Homing</string>
    </property>
    <addaction name="actionRunHoming"/>
    <addaction name="actionOverride"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHoming"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="enableAxis"/>
   <addaction name="actionHalt"/>
  </widget>
  <widget class="QToolBar" name="toolBar_Jog">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBarJog</string>
   </property>
   <property name="toolTip">
    <string>Jog Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionJogPendant"/>
  </widget>
  <widget class="QDockWidget" name="dock_ptp">
   <property name="minimumSize">
    <size>
     <width>287</width>
     <height>178</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Point-to-Point Motion</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QRadioButton" name="rbRelative">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Relative:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbGo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Go (Ctrl+Enter)</string>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
     <property name="icon">
      <iconset resource="icons/tow_resources.qrc">
       <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+Return</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbAbsolute">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Absolute:</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_shortcuts">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>61</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Shortcuts</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="pbGoRightLimit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Go to Right Limit</string>
      </property>
      <property name="icon">
       <iconset resource="icons/tow_resources.qrc">
        <normaloff>:/icons/go-last.png</normaloff>:/icons/go-last.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pbGoPlatform">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Go to Platform</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons/tow_resources.qrc">
        <normaloff>:/icons/go-home.png</normaloff>:/icons/go-home.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pbGoLeftLimit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Go to Left Limit</string>
      </property>
      <property name="icon">
       <iconset resource="icons/tow_resources.qrc">
        <normaloff>:/icons/go-first.png</normaloff>:/icons/go-first.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbHalt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Halt Axis (Ctrl+Backspace)</string>
     </property>
     <property name="text">
      <string>Halt</string>
     </property>
     <property name="icon">
      <iconset resource="icons/tow_resources.qrc">
       <normaloff>:/icons/agt_stop.png</normaloff>:/icons/agt_stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="accSpinBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="velSpinBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.300000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="posSpinBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-26.100000000000001</double>
     </property>
     <property name="maximum">
      <double>26.100000000000001</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocity (m/s):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Target (m):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Acc./Dec. (m/sÂ²):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="traverseOff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="traverse1m">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 m</string>
   </property>
  </action>
  <action name="traverse2m">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 m</string>
   </property>
  </action>
  <action name="traverse3m">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 m</string>
   </property>
  </action>
  <action name="traverse4m">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4 m</string>
   </property>
  </action>
  <action name="actionWiki">
   <property name="text">
    <string>Tow Tank Wiki</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="traverseCustom">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionRunHoming">
   <property name="text">
    <string>Run Homing</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionOverride">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Override</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="enableAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/tow_resources.qrc">
     <normaloff>:/icons/checkmark.png</normaloff>:/icons/checkmark.png</iconset>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable or Disable Axis (Ctrl+E)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionViewTowAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tow Axis</string>
   </property>
  </action>
  <action name="actionViewTraverseAxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Traverse Axes</string>
   </property>
  </action>
  <action name="actionTurbine_Axis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Turbine Axis</string>
   </property>
  </action>
  <action name="actionWave_Maker">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Wave Maker</string>
   </property>
   <property name="toolTip">
    <string>Some day...</string>
   </property>
  </action>
  <action name="actionHalt">
   <property name="icon">
    <iconset resource="icons/tow_resources.qrc">
     <normaloff>:/icons/agt_stop.png</normaloff>:/icons/agt_stop.png</iconset>
   </property>
   <property name="text">
    <string>Halt</string>
   </property>
   <property name="toolTip">
    <string>Halt Axis (Ctrl+Backspace)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionJogPendant">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/tow_resources.qrc">
     <normaloff>:/icons/yellow_power.png</normaloff>:/icons/yellow_power.png</iconset>
   </property>
   <property name="text">
    <string> Pendant</string>
   </property>
   <property name="toolTip">
    <string>Jog Pendant</string>
   </property>
  </action>
  <action name="actionOn_Enable">
   <property name="text">
    <string>On Enable</string>
   </property>
  </action>
  <action name="actionOn_PTP_Move">
   <property name="text">
    <string>On PTP Move</string>
   </property>
  </action>
  <action name="actionOn_Jogging">
   <property name="text">
    <string>On Jogging</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>rbAbsolute</tabstop>
  <tabstop>rbRelative</tabstop>
  <tabstop>posSpinBox</tabstop>
  <tabstop>velSpinBox</tabstop>
  <tabstop>accSpinBox</tabstop>
  <tabstop>pbHalt</tabstop>
  <tabstop>pbGo</tabstop>
  <tabstop>pbGoLeftLimit</tabstop>
  <tabstop>pbGoPlatform</tabstop>
  <tabstop>pbGoRightLimit</tabstop>
  <tabstop>pbJogPendant</tabstop>
  <tabstop>pbJogPlus</tabstop>
  <tabstop>pbJogMinus</tabstop>
 </tabstops>
 <resources>
  <include location="icons/tow_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
